pi-apps-terminal-bash-edition.sh

```diff
@@ -148,106 +167,219 @@ while [ "$1" != "" ]; do
 		;;
 		install)
 			shift
-			for arg in "$@"; do
-				cmdflags+="$arg "
-			done
-			$PI_APPS_DIR/manage install "$(echo $cmdflags)"
-			exit $?
+			if [ -d "$DIRECTORY/apps/$*" ]; then # if only one app is provided
```

Comment:
```suggestion
			if [[ -d "$DIRECTORY/apps/$*" ]]; then # if only one app is provided
```

`[[` is the more feature rich version of `[`, so I prefer using it.